sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/healthcare/HL7v2IOTest.java

```diff
@@ -59,6 +61,30 @@ public void test_HL7v2IO_failedReads() {
     pipeline.run();
   }
 
+  @Test
+  public void test_HL7v2IO_failedReadsByParameter() {
```

Comment:
```suggestion
  public void testHL7v2IOFailedReadsByParameter() {
```

We use smallCamelNamingStyle in Java

This leads to me to wonder if our checkStyle for test code no longer working.

Reply:
So, I've change the names of the tests to be smallCamelNamingStyle in here.